kos/views.py

```diff
@@ -245,6 +245,18 @@ class ResultsView(DetailView):
     model = Year
     template_name = 'kos/results.html'
 
+    def add_places(self, results):
+        current_place = 1
+        previous_last_correct_submission = None
+        results_list = []
+        for i, result_row in enumerate(results):
+            if previous_last_correct_submission != result_row.last_correct_submission:
```

Comment:
Inak to sa týka asi aj MP. ze tam sa nemoze stat nieco take ze ak maju dvaja timy v ten isty cas odovzdanu poslednu sifru ale jeden tim ma o sifu viac tak ich to da na rovnake miesto?

Reply:
Snad opravene, vytvorim aj v MP pr